component/background/RoomService.tsx

```diff
@@ -76,29 +78,29 @@ class RoomService {
     if (!data) throw new Error(`data is {data}`);
     this.messageRef.push().set({
       data,
-      fromID: this.myConfRef.key
+      fromID: this.id
     });
   }
 
-  public getUser(userID: string): Promise<any> {
-    return this.getDataAtReference(this.userListRef.child(userID));
+  public getUser(userID: string): Promise<void> {
+    return this.getDataAtReference(allUsers.child(userID));
   }
 
-  public getMySelf(): Promise<any> {
-    return this.getDataAtReference(this.myConfRef);
+  public getMySelf(): Promise<void> {
+    return this.getDataAtReference(this.myself);
   }
 
   public updateConf(confData: any): void {
-    this.myConfRef.update(confData);
+    this.myself.update(confData);
   }
 
   public setConf(confData: any): void {
-    this.myConfRef.set(confData);
+    this.myself.set(confData);
   }
 
-  public getDataAtReference(reference: firebase.database.Reference): Promise<any> {
-    return new Promise<any>((resolve) => {
-      this.myConfRef.once("value", (data) => {
+  private getDataAtReference(reference: firebase.database.Reference): Promise<void> {
+    return new Promise<void>((resolve) => {
+      this.myself.once("value", (data) => {
```

Comment:
Does `firebase` not specify a fail case? If not, ignore this comment. Otherwise, you should use `reject` so that we can be at least aware of such occasions.